toqito/matrix_ops/tensor.py

```diff
@@ -141,7 +141,7 @@ def tensor(*args) -> np.ndarray:
             result = args[0][0]
             for i in range(1, len(args[0])):
                 result = np.kron(result, args[0][i])
-        return result
+            return result
 
```

Comment:
@vprusso I think these 3 if conditions are not properly defined. The intended input in this case is supposed to be a numpy array inside a list. 

The if condition at L140 is not triggered because the first if at L136 also might be triggered by the same set of conditions. Should I go ahead and remove L140-144?

Reply:
I think so, yeah. If you do, do any of the test cases fail?